### ISSUE TITLE
Asymmetric errors

### ISSUE THREAD

**Issue by BeastyBlacksmith** [JuliaPhysics/Measurements.jl#69]
Are you planning to support asymmetric errors like `2.0 + 0.1 - 0.05`?

**Comment by giordano** [comment]
To be honest, I've never got how this works.  Is the propagation rule the same as with symmetric errors, but using different sigmas on each side?  Do symmetric errors break the assumption that errors are normally distributed?

**Comment by BeastyBlacksmith** [comment]
You are right, it does. Seems like MC is the way to go then.
I found this paper: https://iopscience.iop.org/article/10.1088/1681-7575/ab2a8d/pdf

**Comment by giordano** [comment]
Yeah, this looks more something for [`MonteCarloMeasurements.jl`](https://github.com/baggepinnen/MonteCarloMeasurements.jl) than this package.

**Comment by derikk** [comment]
The [Particle Data Group _Review_](https://pdg.lbl.gov) uses asymmetric errors for many of its consensus particle data values (e.g. [quark masses](https://pdglive.lbl.gov/Particle.action?node=Q123)). These are in turn used by packages like [`Corpuscles.jl`](https://github.com/JuliaPhysics/Corpuscles.jl), which currently implements its own [`MeasuredValue` struct and methods](https://github.com/JuliaPhysics/Corpuscles.jl/blob/bcf2e47fad92aa1022b4e7f2867abdc8b5d8c3b9/src/Corpuscles.jl#L83-L118). This works in some cases, but requires code duplication, and doesn't play nicely with other packages.

Using `MonteCarloMeasurements.jl` for that package would be too heavy-handed, and it's not clear how that would be done without knowing the underlying distribution that the uncertainties come from.

It would be reasonable to expect functions like `stdscore` to error for measurements with asymmetric uncertainties, where the assumption of normality breaks down.

**Comment by derikk** [comment]
On the other hand, given that so much of this package's structure assumes normally distributed errors, perhaps it's best not to make such a big change.

**Comment by AhmedSalih3d** [comment]
Is this still not possible? Any other Julia package which can do this?
